packages/measurement-value/float-cdf.js

```diff
@@ -0,0 +1,42 @@
+const _ = require('lodash');
+
+const { MeasurementValue } = require('./measurement-value');
+
+class FloatCdf extends MeasurementValue {
+  xs(v) {
+    return v === undefined
+      ? _.get(this.input, 'xs')
+      : _.set(this.input, 'xs', v);
```

Comment:
can you use an es6 getter/setter instead?
https://stackoverflow.com/questions/28222276/what-are-getters-and-setters-for-in-ecmascript-6-classes

Reply:
I do not like setter/getters according to my own experience and knowledge they mislead clients of classes. We should write either "getXs()" or "xs()". But do not get properties directly. May this book shows it (https://www.amazon.com/Code-Complete-Practical-Handbook-Construction/dp/0735619670) may be another.

Reply:
There are exceptions but not in this case.

Reply:
Ok, I'm find with that.

Actually, now that I think about it, why do we even need a setter? Can't we just set these when we create a new instance. I assume they really shouldn't need to be mutated.

Reply:
Ok, will remove setter.